src/index.html

```diff
@@ -8,6 +8,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+
+      <div class="stars--row stars--0">
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+        <div class="stars"></div>
+      </div>
+
+      <div class="stars--row stars--1">
+        <div class="stars-active"></div>
```

Comment:
```suggestion
        <div class="stars-star"></div>
```
<img width="467" alt="Screenshot 2023-10-19 at 15 52 23" src="https://github.com/mate-academy/layout_stars/assets/135232001/cca05d08-47a5-4a59-a5f3-af6912b9ef6e">